hi-ml-cpath/environment.yml

```diff
@@ -126,8 +126,10 @@ dependencies:
     - google-auth==1.35.0
     - google-auth-oauthlib==0.4.6
     - grpcio==1.47.0
+    - hi-ml==0.2.3
```

Comment:
If we include this pinned version of hi-ml as a requirement, will there be any problems in the build pipeline, when we initially create the python environment with this pinned package, then call pip locally inside the hi-ml module?

Reply:
(and same for hi-ml-azure)

Reply:
Yes, I checked that this works. If `hi-ml` is already installed, a next attempt to install `hi-ml` from local in editable mode will unistall the existing, and put the local version on top.
An alternative way would have been to exclude `hi-ml` when we build the locked environment.